src/main/resources/db/migration/V1_70__referral_assignment_history.sql

```diff
@@ -0,0 +1,21 @@
+create table referral_assignments
+(
+    referral_id    uuid not null,
+    assigned_at    timestamp with time zone,
+    assigned_by_id text,
+    assigned_to_id text,
+    constraint fk_referral_id foreign key (referral_id) references referral,
+    constraint fk_assigned_by_id_auth_user foreign key (assigned_by_id) references auth_user,
+    constraint fk_assigned_to_id_auth_user foreign key (assigned_to_id) references auth_user
+);
+
+insert into referral_assignments
+select id, assigned_at, assigned_by_id, assigned_to_id
+from referral;
+
+alter table referral
+    drop column assigned_at;
+alter table referral
+    drop column assigned_by_id;
+alter table referral
+    drop column assigned_to_id;
```

Comment:
Should you take a copy of the table before doing this migration so you have a rollback point?

Reply:
as discussed on slack, going to keep the old columns around as suggested by david to achieve this.